jobs/build/ocp4/Jenkinsfile

```diff
@@ -12,8 +12,8 @@ node {
                 logRotator(
                     artifactDaysToKeepStr: '',
                     artifactNumToKeepStr: '',
-                    daysToKeepStr: '',
-                    numToKeepStr: '1000')),
+                    daysToKeepStr: '30',
```

Comment:
Good call. But 60 or 90 days might be better. Personally I have gone back as far as 3 months and ran analytics on build logs before

Reply:
+1 - 60 should be fine